cmd/clusterctl/client/cluster/proxy.go

```diff
@@ -183,6 +212,20 @@ func (k *proxy) ListResources(labels map[string]string, namespaces ...string) ([
 				continue
 			}
 
+			// Continue if the resource is an excluded CRD.
+            gv, err := schema.ParseGroupVersion(resourceGroup.GroupVersion)
```

Comment:
Can you please run gofmt, this indentation is wrong. Not sure why our jobs didn't detect it, but running make lint locally did

Reply:
Looks like GitHub didn't rigger the golangci-lint workflow on this PR (@fabriziopandini @CecileRobertMichon is this something we might have to enable in the repo settings? otherwise it was probably a GitHub flake.

Reply:
TBH I think this is something we forgot to do when creating this branch

Reply:
@sbueringer should we change this (not really sure how github actions work on older branch)? https://github.com/kubernetes-sigs/cluster-api/blob/3f56d7bc00588f384ee6c0ee5c3f7e5a6cb6c196/.github/workflows/golangci-lint.yml#L6

Reply:
@fabriziopandini Maybe it just doesn't :). I would suggest changing that to something which matches our release branches too (https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) and then backporting that change

I think (or more guess) that GitHub uses the workflow config of the base branch of a PR to decide which workflows should be run.

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/5723

Reply:
done